### ISSUE TITLE
Aliasing GPIO names at the board level still available for after v2 Refactor?

### ISSUE THREAD

**Issue by jaredwolff** [atsamd-rs/atsamd#364]
One thing I did notice during my migration here: #363  is whether or not now there's a way to alias the names of the pins for specific boards. Is that still possible or with the refactor that won't work anymore?

**Comment by sajattack** [comment]
The define_pins macro is still available in v2 https://github.com/atsamd-rs/atsamd/blob/f36882efa0708126f59e995530c6420e43bb3da6/hal/src/common/gpio/v2/pin.rs#L1419

**Comment by jaredwolff** [comment]
Thanks @sajattack 

Maybe it's my lack of understanding. Looks like that macro is being used here: https://github.com/atsamd-rs/atsamd/blob/f36882efa0708126f59e995530c6420e43bb3da6/hal/src/common/gpio/v2/pin.rs#L1447

Am I wrong?

I can access them this way. 

```rust
        let mut pins = Pins::new(peripherals.PORT);
        let led_pass = pins.pa23.into_push_pull_output();
```

**Comment by sajattack** [comment]
I asked @bradleyharden about this and he said he hasn't implemented a solution for BSP aliases yet but he's going to work on it.

**Comment by jaredwolff** [comment]
Cool. Thanks @sajattack. I'll close this out unless you guys want to keep this open for tracking purposes.

**Comment by sajattack** [comment]
Might as well keep it open until it's implemented.

**Comment by bradleyharden** [comment]
@jaredwolff, the `v2` `define_pins` macro is actually completely unrelated to the `v1` `define_pins` macro. That's just an unfortunate naming coincidence. The `v2` version is intended only for internal use.

I'm looking at the `v1` `define_pins` macro now, and I think it would make sense to define something equivalent for `v2`. I'll try to put something together that satisfies the same purpose and perhaps improves on it, as discussed in #363.

**Comment by bradleyharden** [comment]
Does #371 satisfy this issue? If it is merged, should this issue be closed?

**Comment by jaredwolff** [comment]
Correcto. We can close this out once that is merged.
